browser/brave_wallet/brave_wallet_service_delegate_impl_android.cc

```diff
@@ -12,12 +12,30 @@
 #include "brave/components/brave_wallet/browser/permission_utils.h"
 #include "brave/components/permissions/contexts/brave_wallet_permission_context.h"
 #include "chrome/browser/profiles/profile.h"
+#include "chrome/browser/ui/android/tab_model/tab_model.h"
+#include "chrome/browser/ui/android/tab_model/tab_model_list.h"
 #include "content/public/browser/browser_context.h"
 #include "content/public/browser/web_contents.h"
 #include "url/gurl.h"
 
 namespace brave_wallet {
 
+namespace {
+content::WebContents* GetActiveWebContents(content::BrowserContext* context) {
+  auto tab_models = TabModelList::models();
+  auto iter = base::ranges::find_if(
+      tab_models, [](const auto& model) { return model->IsActiveModel(); });
+  if (iter == tab_models.end()) {
+    return nullptr;
+  }
+
+  auto* active_contents = (*iter)->GetActiveWebContents();
+  DCHECK(active_contents);
```

Comment:
Let's assume GetActiveWebContents() may return nullptr in production, so return here instead of dcheck